packages/web-shared/components/Searchbar/Search.styles.js

```diff
@@ -84,12 +84,14 @@ const Wrapper = withTheme(styled.div`
     border-radius: 0;
   }
 
-  .chip-list-feature {
-    * > &:first-child {
+  .chip-list-feature ul {
+    overflow-x: scroll;
+
```

Comment:
I think adding `padding-bottom: 1rem;` here might be nice.

Reply:
👁️ I'll take a look!

Reply:
Looks good without the scrollbar, looks not so good with the scrollbar. Margin might look great. 
<img width="477" alt="CleanShot 2024-02-13 at 12 57 46@2x" src="https://github.com/ApollosProject/apollos-embeds/assets/1637694/5667c43c-ce90-4172-ab18-9738ebba1a69">

Reply:
Splitting 1rem across margin + padding

<img width="444" alt="CleanShot 2024-02-13 at 12 59 01@2x" src="https://github.com/ApollosProject/apollos-embeds/assets/1637694/7f32cc13-2616-49de-be91-64f8be711e42">

Just margin

<img width="429" alt="CleanShot 2024-02-13 at 12 59 11@2x" src="https://github.com/ApollosProject/apollos-embeds/assets/1637694/a2daf77f-167c-415c-890f-b0c8c2359b83">

I think Iike splitting the most

Reply:
Yeah Splitting works for me